### PR TITLE
feature/BDD_CODE_IP2_591-List_of-technology-groups

### PR DESCRIPTION
**Pull request by Marta9pa**

### CODE REVIEW

File: RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        {
+            IRestResponse response = client.Execute(request);
+            var responseMessage = JsonConvert.DeserializeObject<Group>(response.Content);
+        
+            Assert.AreEqual(code, (int)response.StatusCode);
+            Assert.That(responseMessage.groups[0], Is.EqualTo("Java"));
+            Assert.That(responseMessage.groups[1], Is.EqualTo("JavaScript"));
+            Assert.That(responseMessage.groups[2], Is.EqualTo("QA"));
+            Assert.That(responseMessage.groups[3], Is.EqualTo("Mobile (Android)"));
+            Assert.That(responseMessage.groups[4], Is.EqualTo("Pearl Advanced"));
```
Comment (js40598) [inline RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs:38]: Would be nice to ensure that test will work even if order of tech groups will change. Like that

            Assert.That(responseMessage.groups.Contains("Java"), Is.True);
            Assert.That(responseMessage.groups.Contains("JavaScript"), Is.True);
            Assert.That(responseMessage.groups.Contains("QA"), Is.True);
            Assert.That(responseMessage.groups.Contains("Mobile (Android)"), Is.True);

You could also check if amount of groups is correct 

            Assert.That(responseMessage.groups.Count, Is.EqualTo(4));

"Pearl Advanced" group has been removed from api.

File: RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    [Binding]
+    public class ListOfTechnologyGroupsSteps
+    {
+        RestClient client = new RestClient();
+        IRestRequest request = new RestRequest();
+       
+        [Given(@"User sets the proper url")]
+        public void GivenUserSetsTheProperUrl()
+        {
+            client = new RestClient("http://intive-patronage.pl:9101");
```
Comment (js40598) [inline RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs:18]: Url has been changed to
http://intive-patronage.pl

**Review by js40598 — CHANGES_REQUESTED** [review]

File: RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs (hunk 1 above)
Comment (Marta9pa) [reply RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs]: That's a good solution, I changed asserts, thanks.

File: RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs (hunk 2 above)
Comment (Marta9pa) [reply RestSharpProject/Steps/ListOfTechnologyGroupsSteps.cs]: Changed.

**Review by js40598 — APPROVED** [review]

**Pull request merged**